doc/source/getting_started/comparison/includes/filtering.rst

```diff
@@ -0,0 +1,18 @@
+:orphan:
+
+DataFrames can be filtered in multiple ways; the most intuitive of which is using
+:ref:`boolean indexing <indexing.boolean>`
+
+.. ipython:: python
+
```

Comment:
i think in all of these you need a `:suppress:` directive that defines tips :->

Reply:
Done. It's a bit of a hack, so let me know if you'd like it done differently.